script/get_catalog.sh

```diff
@@ -21,14 +21,12 @@ rootdir=$location/../
 if [ "$#" -ge 1 ]; then
   runtimeVersion="$1"
   shift 1
-  $rootdir/mvnw -q \
-    -f ${rootdir}/build/maven/pom-catalog.xml \
-    -Dcatalog.path=${rootdir}/resources \
-    -Druntime.version=$runtimeVersion \
-    -B \
-    -U \
-    "$@"
+  # the catalog was already provided by Camel K runtime. No need to depends on maven to rebuild it
+  wget -q https://repo1.maven.org/maven2/org/apache/camel/k/camel-k-catalog/$runtimeVersion/camel-k-catalog-$runtimeVersion-catalog.yaml -O ${rootdir}/resources/camel-catalog-$runtimeVersion.yaml
+  # TODO if we need to use also staging, then we must parse the maven-metadata from the staging repository 
```

Comment:
@oscerd I'd like your opinion here. When we're using stable version, we should be fine picking up the catalog built. Is there any interim situation where we're using `SNAPSHOT` (ie, during release process)? if so, I can work to include it as described in the TODO comment.

Reply:
Thanks. Then, I'll proceed with the other points in the PR and if all is fine I will merge. If it happens anything wrong during the release process I'll handle.